pages/iam/concepts.mdx

```diff
@@ -32,6 +32,14 @@ Previously, an API key was associated with a single Scaleway [Project](#project)
 
 With the introduction of IAM, an API key is now associated with an IAM [user](#user) or [application](#application). This allows fine-grained access to be defined for the IAM user bearing the API key across different Organizations, Projects, and resources.
 
+## Common Expression Language (CEL)
+
+Common Expression Language (CEL) is the expression language used to specify an expression in [conditions](#conditions) within an IAM policy. The language helps express attribute-based logic expressions. In general, a condition expression consists of one or more statements. Each statement expresses an attribute-based control rule, and ultimately determines whether the permissions applies. IAM conditions use the following CEL features: **Variables**, **Operators**, **Functions** and **Logical Operators**. Refer to the [Understanding policy conditions](/iam/reference-content/understanding-policy-conditions) documentation page for a detailed description of the supported CEL features.
+
+## Conditions
+
+A condition is an additional layer of restrictions for your rule. You can allow access to specific user agents, IP addresses and on a given date or time. Conditions are defined through [CEL](#common-expression-language-cel) expressions, and can be set up and configured in the Scaleway console. Refer to the [Understanding policy conditions](/iam/reference-content/understanding-policy-conditions) documentation page to learn how they are set up and how you can define them.
```

Comment:
I feel like the second sentence doesn't quite make sense (used throughout the doc) gramatically.

Maybe "You can allow access to specific user agents or IP addresses, or restrict to certain dates and times." ? Not sure.

Reply:
ah ui haha
c'est fix, merci!